packages/janet.rb

```diff
@@ -0,0 +1,31 @@
+require 'package'
+
+class Janet < Package
+  description 'a functional and imperative programming language.'
+  homepage 'https://janet-lang.org/'
+  version '0.6.0'
+  source_url 'https://github.com/janet-lang/janet/archive/v0.6.0.tar.gz'
+  source_sha256 '1226240f8ed7f5faafbff6d93e5802c7959c4b40f9212ac6f020d67ef3aa599e'
+
+  def self.patch
+    system 'sed -i "s#-ldconfig#/usr/local/sbin/ldconfig#g" Makefile'
```

Comment:
`system "sed -i 's#-ldconfig##{CREW_PREFIX}/sbin/ldconfig#g' Makefile"`?

Reply:
This most likely isn't needed. `ldconfig` can in most cases be skipped altogether because of `LD_LIBRARY_PATH`.